mobile/lib/shared/models/etag.dart

```diff
@@ -5,9 +5,10 @@ part 'etag.g.dart';
 
 @Collection(inheritance: false)
 class ETag {
-  ETag({required this.id, this.value});
+  ETag({required this.id, this.assetCount, this.time});
   Id get isarId => fastHash(id);
   @Index(unique: true, replace: true, type: IndexType.hash)
   String id;
-  String? value;
+  int? assetCount;
```

Comment:
what is this assetCount counts toward?

Reply:
It was previously stored in 'value'. It's used for device albums. If the current assetCount according to Photomanager is different to the stored assetCount in DB, the album is synced.